src/canmatrix/formats/cmjson.py

```diff
@@ -42,6 +42,7 @@ def dump(db, f, **options):
     export_canard = options.get('jsonCanard', False)
     motorola_bit_format = options.get('jsonMotorolaBitFormat', "lsb")
     export_all = options.get('jsonAll', False)
+    native_types = options.get('jsonNativeTypes', False)
```

Comment:
```python
number_converter = float if native_types else str
```
Then below we can just unconditionally
```python
"factor": number_converter(signal.factor)
```
and similar.

Reply:
Sure.